style.css

```diff
@@ -0,0 +1,62 @@
+.inputish {
+  width: 30rem;
+  margin-left: 80px;
+  margin-right: 0%;
+  height: 2rem;
+  margin-bottom: 2%;
+  margin-top: 10px;
+  vertical-align: middle;
+}
+
+.divish {
+  display: inline-block;
+  margin-left: 35%;
+}
+
+.flexito {
+  display: flex;
+  flex-direction: row;
+}
+
+.superflex {
+  display: flex;
+  flex-direction: column;
+}
+
+.miniflex {
+  display: flex;
+}
+
+.ache2 {
+  margin-left: 54%;
+}
+
+.divzito {
+  border-bottom: 1px solid grey;
+  margin-top: 2%;
+  margin-bottom: 2%;
+}
+
+.imagencita1 {
+  width: 2rem;
+  margin-left: 60%;
+}
+
+.imagencita2 {
+  width: 2rem;
+  margin-left: 1%;
+}
+
+.aladito {
```

Comment:
This can be specified in pixel or reduced in its current unit, to avoid the mess it is causing now